Hardware/Hardware/CardReader/StripeCardReader/CardPresentDetails+Stripe.swift

```diff
@@ -0,0 +1,39 @@
+import StripeTerminal
+
+extension CardPresentDetails {
+
+    /// Convenience initializer
+    /// - Parameter reader: An instance of a StripeTerminal.CardPresentDetails
+    init(details: StripeCardPresentDetails) {
+        self.last4 = details.last4
+        self.expMonth = details.expMonth
+        self.expYear = details.expYear
+        self.cardholderName = details.cardholderName
+        self.brand = CardBrand(brand: details.brand)
+        self.fingerprint = details.fingerprint
+        self.generatedCard = details.generatedCard
+        self.receipt = Hardware.ReceiptDetails(receiptDetails: details.receipt)
+        self.emvAuthData = details.emvAuthData
+    }
+}
+
+
+/// The initializers of StripeTerminal.CardPresentDetails are annotated as NS_UNAVAILABLE
```

Comment:
What would you think about asking Stripe to change this in a future version of the SDK?

Reply:
I've taught about it... I think it makes sense, from their point of view, to seal these objects, specially CardReader, so that we can not instantiate our own and pass them back to the SDK. If I try to put myself in their shoes, as a developer of an SDK that integrates with something that takes people's money, I would probably seal my model objects as much as I could. That definitely makes things more difficult for us, in order to unit test things, but it is also fair to admin that we might be trying to do something (unit test everything) that the SDK is not meant for. 

So... I don't know.